src/soundmanager.h

```diff
@@ -62,13 +62,13 @@ class SoundManager : public QObject {
     // open, this method simply runs through the list of all known soundcards
     // (from PortAudio) and attempts to close them all. Closing a soundcard that
     // isn't open is safe.
-    void closeDevices();
+    void closeDevices(bool wait);
```

Comment:
(note, I added that link just as a general discussion of problems that can arise with bare boolean arguments)